DelayedClipboard/tag-security-privacy-delayed-clipboard-rendering.md

```diff
@@ -0,0 +1,100 @@
+### Questions from https://www.w3.org/TR/security-privacy-questionnaire/
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+For standard formats, this feature doesn't expose any information about the source and destination apps during copy-paste operation. However, for web custom formats, since it's restricted to a specific set of apps, during paste it could expose the source app where the user copied the data from. When a format is delay rendered, the system clipboard requests the data for that format during paste. The source app must provide the data for the delay rendered format when it is requested by the system clipboard, else, the paste would fail in the destination app.
```

Comment:
```suggestion
For standard formats, this feature doesn't expose any information about the source and destination apps during copy-paste operation. However, for web custom formats, since some apps may use ecosystem-specific web custom formats, during paste it could expose the source app where the user copied the data from. When a format is delay rendered, the system clipboard requests the data for that format during paste. The source app must provide the data for the delay rendered format when it is requested by the system clipboard, else, the paste would fail in the destination app.
```

Reply:
Done.